charts/recipes-frontend/values.yaml

```
@@ -0,0 +1,11 @@
+nodejs:
+  applicationPort: 1337
+  image: 'hmctspublic.azurecr.io/recipes/frontend:latest'
+  ingressHost: recipes-backend-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+  aadIdentityName: cnp
```

```suggestion
  aadIdentityName: cnp
```

This can be removed now -- will save us one repo in the future when we clean it up everywhere